radish/documentors/__main__.py

```diff
@@ -0,0 +1,211 @@
+
+    -c, --print-to-console
+      print `--single-file` documentaiton to console
+
+    --label-prefix=<label-prefix>
+      label prefix used for various section of the generated docs. Can be used
+      if you need a unique on in your documentation. By default set to
+      `radish.document_id.`
+
+    -r, --recreate
+      recreate output folder or file by removing and creating it again.
+
+    -h, --help
+      show this screen
+
+    -v, --version
+      show version
+
+(C) Copyright by Timo Furrer <tuxtimo@gmail.com>
+    """
+
+    # enable errororacles except hook system wide
+    enable_excepthook()
+
+    # parse sys.argv against our docopt argments
+    arguments = docopt("radish-docs {0}\n{1}".format(__VERSION__, usage),
+                       version=__VERSION__)
+
+    # create world wide configuration
+    setup_config(arguments)
+
+    # process arguments for "sphinx" command
+    if arguments['sphinx']:
+        # get the module path
+        module_path = arguments['<module-path>']
+
+        # get the output folder or if missing set the default for the
+        # output_folder_path
+        if arguments['--output-folder'] is not None:
+            output_folder_path = arguments['--output-folder']
+        else:
+            output_folder_path = "${PWD}/_build"
+
+        # get the index file name, otherwise default it to index.rst
+        if arguments['--index-file-name'] is not None:
+            index_file_name = arguments['--index-file-name']
+            # test if given file name rather then path, raises RadishError
+            _test_file_name(index_file_name, '--index-file-name')
+        else:
+            index_file_name = "index.rst"
+
+        # set recreate file
+        single_file_path = arguments['--single-file']
+        # set print to console flag
+        print_to_console = arguments['--print-to-console']
+
+        # if console is specified then then set single file to none and
+        # output_folder_path to None, recreate to Flase
+        if print_to_console is True:
+            single_file_path = None
+            output_folder_path = None
+            recreate = False
+
+        # set label prefix
+        if arguments['--label-prefix'] is not None:
+            label_prefix = arguments['--label-prefix']
+        else:
+            label_prefix = "radish.document_id."
+
+        # set recreate flag
+        recreate = arguments['--recreate']
+
+        return_code = document_with_sphinx(module_path,
+                                           output_folder_path,
+                                           index_file_name,
+                                           single_file_path,
+                                           recreate,
+                                           print_to_console,
+                                           label_prefix)
+        return return_code
+
+
+# ........................................................................... #
```

Comment:
If you really like those *code separation line comments* you can leave them. If not, remove them :)